woofbot-lnd/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: woofbot-lnd_web_1
```

Comment:
Nice change to container names for docker DNS! Feel free to do the same thing to Woofbot without Lightning for the next update.

Reply:
I'm afraid to change things in the original WoofBot that would break updates...

It will make more sense to have a mechanism for "dynamic permissions", where the user could choose whether to allow access to LND or not during installation (Or maybe not have LND installed at all).
The docker-compose.yml and umbral-app.yml files should be built dynamically, from a script or a template.
For example, if I'll want WoofBot to integrate in the future with the SimpleTorrent app, to get a notification when a download is complete, I'll have to make 4 copies: WoofBot-without-lightning-without-torrents, WoofBot-LND-without-torrents, WoofBot-without-lightning-SimpleTorrent, WoofBot-LND-SimpleTorrent...

Reply:
Ya, I hear you on this. If you ever want to change OG WoofBot, just let me know and I can do all necessary changes/testing to make sure it doesn't break on update.

Reply:
> For example, if I'll want WoofBot to integrate in the future with the SimpleTorrent app, to get a notification when a download is complete, I'll have to make 4 copies: WoofBot-without-lightning-without-torrents, WoofBot-LND-without-torrents, WoofBot-without-lightning-SimpleTorrent, WoofBot-LND-SimpleTorrent...

Agreed, need a way to do this with dynamic permissions. I would love WoofBot with even more functionality.